libwild/src/args.rs

```diff
@@ -91,6 +91,8 @@ pub struct Args {
     pub(crate) got_plt_syms: bool,
     pub(crate) b_symbolic: BSymbolicKind,
     pub(crate) relax: bool,
+    pub(crate) unresolved_symbols: UnresolvedSymbols,
+    pub(crate) error_unresolved_symbols: bool,
```

Comment:
Do you think the code that uses this variable might be easier to read if this were an enum instead of a bool? e.g. `UnresolvedSymbolAction::Warn`, `UnresolvedSymbolAction::Error`.